src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -84,7 +96,7 @@ The following sections show how to manage the access of Okta users to your {{ si
 
 ### Assign a user or group
```

Comment:
can we make sure we highlight the part about assignments not provisioning groups downstream? maybe a note or warning could do